pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -39,15 +42,30 @@ import (
 	"github.com/openshift/origin/pkg/cmd/flagtypes"
 	oadmission "github.com/openshift/origin/pkg/cmd/server/admission"
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
-	"github.com/openshift/origin/pkg/cmd/server/etcd"
 	cmdflags "github.com/openshift/origin/pkg/cmd/util/flags"
 	"github.com/openshift/origin/pkg/cmd/util/pluginconfig"
 	overrideapi "github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride/api"
 	serviceadmit "github.com/openshift/origin/pkg/service/admission"
 )
 
 // AdmissionPlugins is the full list of admission control plugins to enable in the order they must run
-var AdmissionPlugins = []string{"RunOnceDuration", lifecycle.PluginName, "PodNodeConstraints", "OriginPodNodeEnvironment", overrideapi.PluginName, serviceadmit.ExternalIPPluginName, "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "AlwaysPullImages", "ResourceQuota", "SCCExecRestrictions"}
+var AdmissionPlugins = []string{
+	"RunOnceDuration",
+	lifecycle.PluginName,
+	"PodNodeConstraints",
+	"OriginPodNodeEnvironment",
+	overrideapi.PluginName,
+	serviceadmit.ExternalIPPluginName,
+	"LimitRanger",
+	"ServiceAccount",
+	"SecurityContextConstraint",
+	"BuildDefaults",
+	"LimitPodHardAntiAffinityTopology",
```

Comment:
@mfojtik add a follow up to look at the anti-affinity scheduler feature to see if there are other aspects we should be concerned about for multi-tenant